libvast/vast/transform_step.hpp

```diff
@@ -13,55 +13,33 @@
 #include <arrow/record_batch.h>
 #include <caf/expected.hpp>
 
+#include <queue>
+
 namespace vast {
 
+using arrow_batch
+  = std::tuple<vast::id, vast::type, std::shared_ptr<arrow::RecordBatch>>;
+using batch_queue = std::deque<arrow_batch>;
+using slice_queue = std::deque<table_slice>;
+using batch_vector = std::vector<arrow_batch>;
```

Comment:
These aliases should not be made globally available in the namespace `vast`. They are unintuitive and hinder readability.

I would not expect an `arrow_batch` to be anything but an `arrow::RecordBatch`, and the name does not carry that information.